calibration/PARADM/CGINS-PARADM-50336__20200817.csv

```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50336,CC_bsipar_par_scaling,1,

```

Comment:
looks correct, but no calibration file available on vault/alfresco to verify cal date

Reply:
@rtravis-whoi I have now uploaded the calibration files to Alfresco (minus the ADCPAM docs since there is not a space for those in the usual calibration docs directories)

[Configuration](https://alfresco.oceanobservatories.org/alfresco/d/d/workspace/SpacesStore/685cd1eb-bd2c-419f-84f2-03b4c8956a37/8304-61003-917-00001_Configuration_Record.xlsx)

[PARAD](https://alfresco.oceanobservatories.org/alfresco/d/d/workspace/SpacesStore/4ca62f3b-04ae-4d3c-8443-59837f3ff432/PARAD-M_QSP2155_SN_50336_Calibration_2020-08-17.pdf)
[DOSTA](https://alfresco.oceanobservatories.org/alfresco/d/d/workspace/SpacesStore/516ade1a-7178-4c30-9594-2e21f89bcb48/DOSTA-M_Optode-4831_SN_891_Calibration_2020-06-14.pdf)
[FLORT](https://alfresco.oceanobservatories.org/alfresco/d/d/workspace/SpacesStore/52376cfe-49f5-4100-be35-7dcb0e048084/FLORT-M_FLBBCDSLC_SN_6744_Calibration_2021-03-18.pdf)
[CTDGV](https://alfresco.oceanobservatories.org/alfresco/d/d/workspace/SpacesStore/e569c30f-105f-428a-9634-2d549a73f8b5/CTDGV-M_SBE-Slocum_SN_9692_Calibration_2021-02-03.pdf)

Reply:
confirmed - looks good